compatibility/sandbox-migration/runner/Migration/Types.hs

```diff
@@ -42,4 +46,33 @@ data Tuple2 a b = Tuple2
   , _2 :: b
   } deriving (Eq, Generic, Show)
 
-instance (A.FromJSON a, A.FromJSON b) => A.FromJSON (Tuple2 a b)
\ No newline at end of file
+instance (A.FromJSON a, A.FromJSON b) => A.FromJSON (Tuple2 a b)
+
+data TemplateId = TemplateId
```

Comment:
Is it worth mentioning in a comment why we don't have the package identifier in here?

Reply:
Added a comment that this is only for testing and we therefore omit the package id for convenience.